src/powerlaces/boot_figreload.clj

```diff
@@ -90,7 +90,7 @@
   the fileset."
   [tmpfiles]
   (mapv #(-> {}
-             (assoc :build-id (util/build-id (:path %)))
+             (assoc :build-id (-> % b/tmp-file .getName util/build-id))
```

Comment:
Hmm, why is only the name used? This way files would clash if there are e.g. a/main.cljs.edn and b/main.cljs.edn?